pkg/nodeconfig/nodeconfig.go

```diff
@@ -588,20 +587,38 @@ func (nc *nodeConfig) UpdateKubeletClientCA(contents []byte) error {
 	return nil
 }
 
-// ensureTrustedCABundle gets the trusted CA ConfigMap and ensures the cert bundle on the instance has up-to-date data
-func (nc *nodeConfig) ensureTrustedCABundle() error {
-	trustedCA := &core.ConfigMap{}
+// SyncTrustedCABundle builds the trusted CA ConfigMap from image registry certificates and the proxy trust bundle
+// and ensures the cert bundle on the instance has up-to-date data
+func (nc *nodeConfig) SyncTrustedCABundle() error {
+	caBundle := ""
+	var cc mcfg.ControllerConfig
 	if err := nc.client.Get(context.TODO(), types.NamespacedName{Namespace: nc.wmcoNamespace,
-		Name: certificates.ProxyCertsConfigMap}, trustedCA); err != nil {
-		return fmt.Errorf("unable to get ConfigMap %s: %w", certificates.ProxyCertsConfigMap, err)
+		Name: "machine-config-controller"}, &cc); err != nil {
```

Comment:
Lets use a variable for `machine-config-controller` if it is being used within the controllerconfig controller as well